src/ParquetMsg.chpl

```diff
@@ -1237,11 +1224,12 @@ module ParquetMsg {
       var sections_sizes_int: [0..#ncols] int; // only fill in sizes for int, uint segarray columns
       var sections_sizes_real: [0..#ncols] int; // only fill in sizes for float segarray columns
       var sections_sizes_bool: [0..#ncols] int; // only fill in sizes for bool segarray columns
-      forall (i, column) in zip(0..#ncols, sym_names) {
+      forall (i, column, ot) in zip(0..#ncols, sym_names, col_objTypes) { // TODO - should we make locals of the arrays?
```

Comment:
The arrays aren't distributed and I believe that only applies to unbounded ranges based on this PR, #1880.

Reply:
it seems like that PR is saying that if the leading iterator is unbounded, chpl will complain about a size mismatch (because `0..` is infinite and `1..5` has only 5 things)

But if the leading iterator is a bounded range it will still be parallel but not distributed. It's not super relevant in this case since none of the other arrays are distributed but def worth keeping in mind. This comment goes into a good deal more detail about this:
https://github.com/Bears-R-Us/arkouda/pull/2159#discussion_r1113699483

the relevant section being:
> The current code:
> ```chpl
> forall (i, (up, low)) in zip(0..#s, hashes) {
>     upper.a[i] = up;
>     lower.a[i] = low;
> }
> ```
> actually turns into a parallel loop that only executes on locale 0 because ranges aren't distributed. So a task per core on locale 0 will be executing, but the rest of the machine will be unused. Then for elements that live on locale 1 for example it's actually something like:

Reply:
I think this is great for documenting everything in one place. This is good to be aware of moving forward as I initially only remembered the case of the unbounded range.